pkg/kubelet/qos/policy.go

```diff
@@ -80,3 +114,12 @@ func GetContainerOOMScoreAdjust(pod *v1.Pod, container *v1.Container, memoryCapa
 	}
 	return int(oomScoreAdjust)
 }
+
+func isSidecarContainer(pod *v1.Pod, container *v1.Container) bool {
+	for _, initContainer := range pod.Spec.InitContainers {
```

Comment:
please add a comment here saying that we need this funny loop because we do not have information about the container type in this function.

I am generally fine to keep it this way. We may need to consider passing container type as an argtument in future. But it will touch so much code that may not worth it now.

Reply:
Done